_inc/client/components/jetpack-notices/index.jsx

```diff
@@ -154,7 +154,7 @@ export const DevModeNotice = React.createClass( {
 				devModeType += __('your site URL lacking a dot (e.g. http://localhost).');
 			}
 
-			const text = __('Currently in {{a}}Development Mode{{/a}} VIA ' + devModeType + '{{br/}}Some features are disabled.',
+			const text = __('Currently in {{a}}Development Mode{{/a}} Via ' + devModeType + '{{br/}}Some features are disabled.',
```

Comment:
I know that this PR was not meant to fix this, but let's do this anyway. We should use the full text every time we call the translation function, there should be no concatenation here. Even if that means that `Currently in...' part will be repeated several times.